src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/ReferenceCodeService.kt

```diff
@@ -7,5 +7,8 @@ import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.mapping.toMod
 
 @Service
 class ReferenceCodeService(private val referenceCodeRepository: ReferenceCodeRepository) {
-  fun getReferenceDataByGroup(groupCode: String, enabled: Boolean): List<ReferenceCode> = referenceCodeRepository.findAllByGroupCodeEquals(groupCode).filter { !enabled || it.enabled }.toModel()
+  fun getReferenceDataByGroup(groupCode: String, enabled: Boolean): List<ReferenceCode> = run {
+    referenceCodeRepository.findAllByGroupCodeEquals(groupCode).filter { !enabled || it.enabled }
+      .sortedBy { it.displaySequence }.toModel()
```

Comment:
Decided to sort at the API level.

We do have the option to do this in the UI as well as the display sequence is included in the returned reference code models.